src/Kentico.Xperience.Siteimprove/Services/ISiteimproveService.cs

```diff
@@ -0,0 +1,46 @@
+﻿using CMS;
+using CMS.Core;
+
+using Kentico.Xperience.Siteimprove;
+
+[assembly: RegisterImplementation(typeof(ISiteimproveService), typeof(SiteimproveService), Lifestyle = Lifestyle.Singleton, Priority = RegistrationPriority.SystemDefault)]
```

Comment:
We should only use this kind of service registration if we have to.

Since we have the built-in ASP.NET Core container available to us, these services should be registered in your `IServiceCollection` extension method.